cmd/antrea-agent/agent.go

```diff
@@ -175,6 +175,24 @@ func run(o *Options) error {
 		encryptionMode = config.TrafficEncryptionModeIPSec
 	}
 	_, ipsecAuthenticationMode := config.GetIPsecAuthenticationModeFromStr(o.config.IPsec.AuthenticationMode)
+
+	var mtuDeduction int
```

Comment:
My earlier point is I want not MTU calculation and Mulit-cluster logics to be any multiple files.

After more thinking, probably let us do this:

-  Add a func to node_config.go, called `CalculateMTUDeduction`, which calculates and returns mtuDeduction with encapMode, tunnelType, and enableMulticlusterGW.
- Change `getNodeMTU()` to decide interface MTU with `NetworkConfig.MTUDeduction`.

Make sense to you?

Reply:
As commented in https://github.com/antrea-io/antrea/pull/4407#discussion_r1065143965, I would also:
> add NetworkConfig.InterfaceMTU and change loadNetworkConfig() to use that, and also remove NodeConfig.NodeMTU.

> The reason is that we want not two ways/structs to decide MTUs, which will be harder to understand and maintain.

Then probably we should rename `getNodeMTU` to `getInterfaceMTU`.

Reply:
Done